install.sh

```diff
@@ -241,22 +241,22 @@ nvm_check_global_modules() {
 
   if [ "${MODULE_COUNT}" != '0' ]; then
     cat <<-'END_MESSAGE'
-	=> You currently have modules installed globally with `npm`. These will no
-	=> longer be linked to the active version of Node when you install a new node
-	=> with `nvm`; and they may (depending on how you construct your `$PATH`)
-	=> override the binaries of modules installed with `nvm`:
+    => You currently have modules installed globally with `npm`. These will no
```

Comment:
https://travis-ci.org/creationix/nvm/jobs/171280410 says that `<<-` can only use hard tabs, so this appears to be necessary?